### DISCUSSION TITLE
What is the symbolic path name in Cosmos

### DISCUSSION THREAD

**Discussion by ghost** [CosmosOS/Cosmos#2561] [Q&A]
I don't know the Path of Cosmos's drive, windows will be C:\\, so what about Cosmos [ Sorry if my English is a bit bad ]

![image](https://user-images.githubusercontent.com/85250803/213343724-31ad64e7-8bf4-41db-aeac-d09dd87ad354.png)
![image](https://user-images.githubusercontent.com/85250803/213343756-b77a6a6c-7a08-4862-95f3-6a2e33f84816.png)

You don't have to worry too much about XConsole which is a class just for Write Text like Console.WriteLine(). The main part is cosmosVFS.CreateFile("0:\\Username.txt"), on the internet most of them use 0:/ or 0:\\, but neither of those methods are feasible for me

**Comment by quajak** [comment]
The first filesystem is currently at 0:\\, second at 0:\\ ...
